wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss

```diff
@@ -1,3 +1,7 @@
+.card {
+	padding: 1.75rem;
+}
```

Comment:
This padding is declared below on line 11. The bit we're missing which affects the course cards on the home page is 

```
body:not(.single):not(.search) .site-main .card.post.course {
	padding: 1.75rem;
}
```

![Screen Shot 2022-10-18 at 11 11 10 AM](https://user-images.githubusercontent.com/1017872/196293326-a95034b8-6074-4dfc-b3a5-667bce3fd7cb.jpg)

Doesn't feel quite right for these styles to be added here, as it's an override for the sensei styles, so a comment is probably necessary, maybe:

```suggestion
// Override Sensei course styles for cards on home page
.body:not(.single):not(.search) .site-main .card.post.course {
	padding: 1.75rem;
}
```

Reply:
That should be fixed now